README.Rmd

```diff
@@ -12,7 +12,7 @@ knitr::opts_chunk$set(
 )
 ```
 
-# tibble <img src="man/figures/logo.png" align="right" />
+# tibble <img src="man/figures/logo.png" align="right" alt="" />
```

Comment:
Do we need an alt text here?

```suggestion
# tibble <img src="man/figures/logo.png" align="right" alt="Hex logo for the tibble package" />
```

Reply:
No, the empty one allows screenreaders to skip it (since it's purely decorative) https://github.com/r-lib/pkgdown/issues/2288#issue-1619790044